Day-I-Social-Card/social-card/src/components/HeaderComponents/HeaderContainer.js

```diff
@@ -1,2 +1,19 @@
 import React from 'react';
 import './Header.css';
+import Logo from './ImageThumbnail';
+import HeaderTitle from './HeaderTitle';
+import HeaderContent from './HeaderContent';
+
+const HeaderContainer = () => {
+    return (
+        <header>
+            <Logo />
+            <div className="headerContent">
+                <HeaderTitle />
+                <HeaderContent />
```

Comment:
You can just put the `HeaderTitle` into the `HeaderContent` to save yourself from having to handle more than one component.